api/src/main/java/org/openmrs/module/openconceptlab/client/OclClient.java

```diff
@@ -125,8 +133,45 @@ public GetMethod executeExportRequest(String url, String latestVersion) throws I
 		return exportUrlGet;
 	}
 
+	public GetMethod getExportUrl(String url, String latestVersion){

```

Comment:
Can you add some javadoc to these methods?

Reply:
done